decksite/data/deck.py

```diff
@@ -615,6 +615,11 @@ def nwdl_join() -> str:
             ) AS dsum ON d.id = dsum.id
     """
 
+def num_decks() -> int:
+    sql = 'SELECT COUNT(DISTINCT id) AS c FROM deck'
+    r = Container(db().select(sql)[0])
+    return r.c
```

Comment:
You can replace these two lines with:

    return db().value(sql)

which gets the value of the only column of the only row of your resultset.